examples/bzlmod/extensions.bzl

```diff
@@ -0,0 +1,39 @@
+"""
+This module is for setting the pyhton_interpreter_target argument
```

Comment:
Can we move this into rules_python instead? I don't think users should have to write their own extension for this. Having to call an extra extension between getting the interpreters and setting up pip is annoying, but far better than having to write your own extension.

Reply:
I don't know how to.  We depend on the python repo that we loaded with use_repo.  That is providing the interpreter.  So with the current code base, the answer is no.  If I understood the hub and spoke model more, maybe ...